### PR TITLE
adding moa faq

### PR DESCRIPTION
**Pull request by lamek**
Adding the MOA faq and installation faqs.

Manual preview is available here - http://file.bos.redhat.com/klamenzo/moa/openshift-moa/moa/getting_started_moa/moa-faq.html

### CODE REVIEW

File: modules/moa-faq-installing.adoc [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+----
+
+Run the following command to generate uninstall logs. Replace <my_cluster_name> with the name of your cluster.
+
+[source,terminal]
+----
+$ moactl logs uninstall <my_cluster_name>
+----
+
+== Verify your AWS account does not have an SCP
```
Comment (jeremyeder) [inline modules/moa-faq-installing.adoc:26]: Prior to first tuse of an acryonym, use expanded version?

File: modules/moa-faq-installing.adoc [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+If you receive any errors, double check to ensure than an link:https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_type-auth.html#orgs_manage_policies_scp[SCP] is not applied to your AWS account. If you are required to use an SCP, see link:https://www.openshift.com/dedicated/ccs#scp[Red Hat Requirements for Customer Cloud Subscriptions] for details on the minimum required SCP.
+
+== Verify your AWS account and quota
+
+Run the following command to verify you have the available quota on your AWS account. 
+
+[source,terminal]
+----
+$ moactl verify quota
```
Comment (jeremyeder) [inline modules/moa-faq-installing.adoc:43]: Here perhaps link to the output of https://issues.redhat.com/browse/OSDOCS-1444 and https://issues.redhat.com/browse/SDA-2838

File: modules/moa-faq-installing.adoc [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+----
+
+If you need to increase your quota, navigate to your link:https://aws.amazon.com/console/[AWS console], and request a quota increase for the service that failed.
```
Comment (jeremyeder) [inline modules/moa-faq-installing.adoc:46]: AWS quota requests are typically tied to a specific region.  Make sure you are requesting quota for the region you intend to use for installation.

**Review by jeremyeder — APPROVED** [review]

File: modules/moa-faq-installing.adoc [hunk 4]
```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// getting_started_moa/moa-faq.adoc
+
+
+[id="moa-faq-installing"]
```
Comment (adellape) [inline modules/moa-faq-installing.adoc:6]: Every heading and subheading in a module should have a unique ID with a `_{context}` suffix. E.g.:

```
[id="moa-faq-installing_{context}"]
```

File: modules/moa-faq-installing.adoc [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+= Installation troubleshooting
+
+== Inspect install or uninstall logs
+
+Run the following command to display install logs. Replace <my_cluster_name> with the name of your cluster.
```
Comment (adellape) [inline modules/moa-faq-installing.adoc:12]: Markup `<my_cluster_name>` in backticks.

File: modules/moa-faq-installing.adoc [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+----
+
+To watch the logs while your cluster is installing, include the `--watch` flag.
+
+[source,terminal]
+----
+$ moactl logs install <my_cluster_name> --watch
+----
+
+Run the following command to display uninstall logs. Replace <my_cluster_name> with the name of your cluster.
```
Comment (adellape) [inline modules/moa-faq-installing.adoc:26]: Markup `<my_cluster_name>` in backticks.

File: modules/moa-faq-installing.adoc [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+= Installation troubleshooting
+
+== Inspect install or uninstall logs
+
+Run the following command to display install logs. Replace <my_cluster_name> with the name of your cluster.
+
+[source,terminal]
+----
+$ moactl logs install <my_cluster_name>
+----
```
Comment (adellape) [inline modules/moa-faq-installing.adoc:17]: General comment for all the procedures, but could most of them be written as a one-step unordered list? E.g.:

```
To display install logs:

* Run the following command, replacing `<my_cluster_name>` with the name of your cluster:
+
[source,terminal]
----
$ moactl logs install <my_cluster_name>
----

Comment (adellape) [reply modules/moa-faq-installing.adoc:17]: Also in general suggest ending the sentences leading into the code blocks with `:`.

**Review by adellape — COMMENTED** [review]
A few minor things!

File: modules/moa-faq-installing.adoc (hunk 2 above)
Comment (lamek) [reply modules/moa-faq-installing.adoc:63]: I'm gonna track this on the quota doc card as I've got to take a second pass at it.
https://issues.redhat.com/browse/OSDOCS-1444?focusedCommentId=15203223&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-15203223

**Pull request merged**